src/modules/Dashboard/features/Participant/Assignments/AboutParticipant/AboutParticipant.test.tsx

```diff
@@ -27,6 +30,7 @@ import { useFeatureFlags } from 'shared/hooks/useFeatureFlags';
 import { Roles } from 'shared/consts';
 
 import AboutParticipant from './AboutParticipant';
+import { ParticipantWithDataAccess } from '../../../../types';
```

Comment:
I just find this depth of relative path a little hard to parse…
```suggestion
import { ParticipantWithDataAccess } from 'modules/Dashboard/types';
```

Reply:
Thanks 🙏  I really wish I could figure out how to get IntelliJ to do this automatically. It always uses relative path. I try to catch them and correct by hand, but I must've missed this one

Reply:
Oh, I think I figured it out. Hopefully we'll have less of these